tests/network/services.go

```diff
@@ -114,20 +114,17 @@ var _ = SIGDescribe("Services", func() {
 			servicePort        = 1500
 		)
 
-		createVMISpecWithBridgeInterface := func(vmi *v1.VirtualMachineInstance) *v1.VirtualMachineInstance {
-			if vmi == nil {
-				vmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n")
-			}
-			vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
-			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultBridgeNetworkInterface()}
-
+		createVMISpecWithBridgeInterface := func() *v1.VirtualMachineInstance {
+			vmi := libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()))
```

Comment:
Do we need this or is it created by default if not specified?

We also do not really need this wrapping function anymore.

Reply:
It is not created by default. I want a VMI without interfaces or networks by default then my tiny helpers can create a VMI tailored to whatever wacky use case the test at hand required. 

Hence I have the helper that you're eagerly trying to get rid of. I think it makes everything more readable, hence sticking with it.

Reply:
The "DefaultPodNetwork" is not created implicitly if not specified?

Reply:
It might, depends on the value of `vmi.Spec.Domain.Devices.AutoattachPodInterface`, as seen on https://github.com/kubevirt/kubevirt/blob/48562f9ef7d0c1dbee92a0a3a5f878346da75350/staging/src/kubevirt.io/client-go/api/v1/defaults.go#L145 . 

I prefer to be explicit.